docs/development-and-testing.md

```diff
@@ -53,6 +53,97 @@ $ ./kubectl-directpv --kubeconfig <PATH-TO-KUBECONFIG-FILE> info
 $ ./kubectl-directpv --kubeconfig <PATH-TO-KUBECONFIG-FILE> drives list
 ```
 
+## Testing with minikube
+
+1. Setup LVs
+
+The following script will create 4 LVs backed up by 4 loopback devices
+
+```bash
+sudo truncate --size=1G /tmp/disk-{1..4}.img
+for disk in /tmp/disk-{1..4}.img; do sudo losetup --find $disk; done
+devices=( $(for disk in /tmp/disk-{1..4}.img; do sudo losetup --noheadings --output NAME --associated $disk; done) )
+sudo pvcreate "${devices[@]}"
+vgname="vg0"
+sudo vgcreate "$vgname" "${devices[@]}"
+for lvname in lv-{0..3}; do sudo lvcreate --name="$lvname" --size=800MiB "$vgname"; done
+```
+
+2. Start minikube
+
+```bash
+minikube start --driver=none
```

Comment:
in some scenario `export CHANGE_MINIKUBE_NONE_USER=true` was required. could you add some insights?